src/open_inwoner/cms/banner/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from cms.models import CMSPlugin
+from filer.fields.image import FilerImageField
+
+
+class Banner(CMSPlugin):
+    title = models.CharField(_("title"), max_length=250, default="")
+    image = FilerImageField(
+        verbose_name=_("Banner image"),
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
```

Comment:
Is `SET_NULL` what we want here? I'd expect `PROTECT` so you can't accidentally remove the homepage image.